content/_data/changelogs/weekly.yml

```diff
@@ -1289,6 +1289,69 @@
     # pull: 3055 too minor (test fix)
     # pull: 3063 too minor (test fix)
     # pull: 3070 too minor (suppress test)
+- version: "2.86"
+  date: 2017-10-22
+  changes:
+    - type: major rfe
+      pull: 3076
+      message: >
+        <em>Launch agent via execution of command on the master</em> has been moved to a new <em>Command Launcher</em> plugin and integrated with the <em>Script Security</em> plugin.
+      references:
+        - issue: 47393
+        - url: https://jenkins.io/security/advisory/2017-10-11/#arbitrary-shell-command-execution-on-master-by-users-with-agent-related-permissions
+          title: related security advisory
+    - type: rfe
+      issue: 36282
+      pull: 3081
+      message: >
+        Add link to recursive <code>cc.xml</code> output on build history page.
+    - type: major bug
+      issue: 47448
+      pull: 3093
+      message: >
+        Fix <em>Download from java.sun.com</em> installation method for JDK for downloads requiring an Oracle login.
+    - type: bug
+      pull: 3097
+      issue: 47500
+      message: >
+        <code>Secret</code> threw <code>ArrayIndexOutOfBoundsException</code> trying to decrypt <code>{}</code>.
+    - type: bug
+      issue: 47455
+      pull: 3090
+      message: >
+        Race conditions in agents going offline could result in an exception when picking a workspace for a build.
+    - type: bug
+      issue: 35459
+      pull: 2994 # and 3089
+      message: >
+        Prevent duplicated elements with incorrect URL when using the search on Dashboard View plugin based views.
+    - type: bug
+      pull: 3099
+      issue: 47517
+      message: >
+        <code>StackOverflowError</code> thrown under some conditions when using Pipeline on 2.85.
```

Comment:
Add "(regression in 2.84)"

Reply:
It's in 2.85, and the message already mentions it.